src/mesh_analysis.rs

```diff
@@ -144,6 +147,78 @@ pub fn triangulated_mesh_genus(vertex_count: usize, edge_count: usize, face_coun
     1 - (cast_i32(vertex_count) - cast_i32(edge_count) + cast_i32(face_count)) / 2
 }
 
+/// Checks if two geometries are visually identical.
+///
+/// The two geometries can
+/// possibly contain the same vertices and normals bound to the same faces, but all
+/// of these can be in different order, including shifts in face winding. This
```

Comment:
> ... including shifts in face winding.

I understood this comment that we consider geometries with opposite winding still identical, a long as they are consistent, but the code doesn't seem to support that. Or?

Reply:
no. a face with an opposite winding is a different face. "shifted winding" means (1, 2, 3) = (2, 3, 1) = (3, 1, 2) and that is still the same face

Reply:
I understand, I just meant that if I didn't know it beforehand, the doc text wouldn't tell me :)